tests/filter_task_by_state.yml

```diff
@@ -0,0 +1,56 @@
+name: Filter Task by State Job
+description: Job to test the filter task by state feature
+service: TES
+versions:
+  - 1.1.0
+tags:
+  - logical
+  - filter
+  - filter task by state test
+  - all
+jobs:
+  - name: create_task
+    description: Create a new TES task
+    endpoint: /tasks
+    operation: POST
+    request_body: |
+      {
+        "name": "ComplianceTest",     
+        "description": "Compliance Test",
+        "executors": [
+          {
+            "image": "alpine",
+            "command": [
+              "echo",
+              "hello"
+            ]
+          }
+        ]
+      }
+    storage_vars:
+      id: $response.id
+    response:
+      200:
+  - name: get_task
+    description: Retrieve the task details for TES task
+    endpoint: /tasks/{id}
+    operation: GET
+    query_parameters:
+      - view: MINIMAL
+    polling:
+      interval: 10
+      timeout: 3600
+    response:
+      200:
+  - name: list_tasks
```

Comment:
Note that this would fail if someone else submitted a job in the meantime. Why not use a `GET /tasks/{id}` call with state `COMPLETE` again? Also, TES doesn't support a "filter tasks by state" feature, as far as I know - so I'm not sure what this test adds, other than to demonstrate what `filter` can do.

Reply:
"filter tasks by state" is added in v1.1.0 in `ListTasks`.
The purpose of creating a task here is to prevent an empty get task list. Even if someone else submitted a job, regardless its state, the expected response should contain tasks with `COMPLETE` state.

Reply:
Oh, I wasn't aware this was added. How do you specify the filter? Didn't see it in the query params

Reply:
The `state` query param was added.
https://github.com/ga4gh/task-execution-schemas/blob/1df37d34242f74d3f03475c6b9de3324b8094054/openapi/task_execution_service.openapi.yaml#L106